__tests__/app.test.js

```diff
@@ -276,53 +303,97 @@ describe("app.js", () => {
           });
         });
     });
-  });
 
-  describe("app.js error handling", () => {
-    test("Status 404: returns an error if path doesn't exist", () => {
+    test("Status 404: returns a message 'article_id not found' if article_id doesn't exist", () => {
       return request(app)
-        .get("/api/topicsbanana")
+        .get("/api/articles/999999/comments")
         .expect(404)
         .then((response) => {
-          expect(response.body.msg).toBe("invalid endpoint");
+          expect(response.body.msg).toBe("article_id not found");
         });
     });
 
-    test("Status 400: returns a message with a value of 'article_id is not a number' if article_id is not a number", () => {
+    test("Status 404: returns a message with 'specified article_id has no comments' for article_id 2", () => {
       return request(app)
-        .get("/api/articles/banana")
-        .expect(400)
+        .get("/api/articles/2/comments")
+        .expect(404)
         .then((response) => {
-          expect(response.body.msg).toBe("article_id is not a number");
+          expect(response.body.msg).toBe(
+            "specified article_id has no comments"
+          );
         });
     });
+  });
 
-    test("Status 404: returns a message 'article_id not found' if article_id doesn't exist", () => {
+  describe("POST requests on /api/articles/:article_id/comments", () => {
+    test("Status 201: returns the inserted comment", () => {
       return request(app)
-        .get("/api/articles/10000")
+        .post("/api/articles/3/comments")
+        .send({
+          username: "icellusedkars",
+          body: "A-ha! Guess who's back in the big-time?",
+        })
+        .expect(201)
+        .then((response) => {
+          expect(response.body.commentInserted).toHaveProperty("comment_id");
+          expect(response.body.commentInserted.comment_id).toBe(19);
+          expect(response.body.commentInserted).toHaveProperty("body");
+          expect(response.body.commentInserted.body).toBe(
+            "A-ha! Guess who's back in the big-time?"
+          );
+          expect(response.body.commentInserted).toHaveProperty("article_id");
+          expect(response.body.commentInserted.article_id).toBe(3);
+          expect(response.body.commentInserted).toHaveProperty("author");
+          expect(response.body.commentInserted.author).toBe("icellusedkars");
+          expect(response.body.commentInserted).toHaveProperty("votes");
+          expect(response.body.commentInserted.votes).toBe(0);
+          expect(response.body.commentInserted).toHaveProperty("created_at");
+        });
+    });
+
+    test("Status 404: returns an error if post request made to a non-existent article_id", () => {
+      return request(app)
+        .post("/api/articles/999999/comments")
+        .send({
+          username: "A. Partridge",
+          body: "A-ha! Guess who's back in the big-time?",
+        })
```

Comment:
make sure that we are only testing one error at a time. We cannot be sure that the 99999 is the error here since the username should also cause an error, since it doesnt exist. Change this for your other error tests too